### PR TITLE
Skravtsov

### PR DESCRIPTION
**Pull request by AlexDyakov**
Added 2 new files

### CODE REVIEW

File: new-file1 [hunk 1]
```diff
@@ -0,0 +1 @@
+Новый файл 1
```
Comment (AlexDyakov) [inline new-file1:1]: Added file 1

File: new-file2 [hunk 2]
```diff
@@ -0,0 +1 @@
+Новый файл2
```
Comment (AlexDyakov) [inline new-file2:1]: Added file2

**Review by AlexDyakov — COMMENTED** [review]
Approve

**Review by AlexDyakov — COMMENTED** [review]
Approved

**Review by kravzo — COMMENTED** [review]
Reviewed

**Review by kravzo — COMMENTED** [review]
Reviewed

**Review by kravzo — APPROVED** [review]
Revieved

**Pull request merged**